inscriptions/ISic000384.xml

```diff
@@ -189,26 +194,31 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141195" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141195" xml:lang="la" resp="#JH">
                 <ab>
                     <lb n="0"/>
-                    <gap reason="lost" extent="unknown" unit="line"/>
+                        <gap reason="lost" extent="unknown" unit="line"/>
                     <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    sibi
+                        <gap reason="lost" extent="unknown" unit="character"/>
+                        sibi
+                        <gap reason="lost" extent="unknown" unit="character"/>
                     <lb n="2"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>is fecit
+                        <gap reason="lost" extent="unknown" unit="character"/>is · fecit
                     <lb n="3"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    suis.
+                        <gap reason="lost" extent="unknown" unit="character"/>
+                        suis
+                        <gap reason="lost" extent="unknown" unit="character"/>
                 </ab>
                 <!--This EDR text 141195 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text of CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#JH">
```

Comment:
all for enough, but generally I'm not sure I'd bother with a text that's so fragmentary ;-)